goltsmannn2/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15...3.30)
+project(new_project
+        VERSION 1.0
+        DESCRIPTION "New project"
+        LANGUAGES CXX)
+add_executable(MyFile src/main.cpp)
+add_library(MyLibrary src/main.cpp
+        src/hello_world.h)
```

Comment:
While technically you have made your library, the intention was that your library would contain purely library code, aka the implementation of the header for hello world. Your main / executable should only call the library. Right now technically your library and executable are the same since your library is header only. Please implement them separately.